hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -476,11 +476,13 @@ int OtbnModel::check() const {
     return -1;
   }
 
-  try {
-    good &= check_call_stack(*iss);
-  } catch (const std::exception &err) {
-    std::cerr << "Failed to check call stack: " << err.what() << "\n";
-    return -1;
+  if (!disable_stack_check_flag) {
```

Comment:
If applying my other suggestion, this changes to
```suggestion
  if (stack_check_enabled_) {
```